### PR TITLE
Fix unit test `conflicts.add_two`

### PR DESCRIPTION
**Pull request by theohax**
There's no open issue for this one as it wasn't disabled.

### CODE REVIEW

File: nano/core_test/conflicts.cpp [hunk 1]
```diff
@@ -54,92 +54,87 @@ TEST (conflicts, add_two)
 	nano::system system{};
 	auto const & node = system.add_node ();
 
+	system.wallet (0)->insert_adhoc (nano::dev::genesis_key.prv);
```
Comment (dsiganos) [inline nano/core_test/conflicts.cpp]: This is a big test. An overview describing what is happening and why would be useful.

Comment (theohax) [reply nano/core_test/conflicts.cpp]: Let's tackle overviews with everyone else.

I'll be pushing a few more test fixes but unless the description of the test is trivial, without such description for now.

**Comment by theohax** [comment]
@reviewers -- ready for final review/merge as well.

**Comment by theohax** [comment]
`wallet.process_block` failed in the CI, it's already on the to-fix list. Will rebase this to fix conflicts and prepare it for merging.

**Comment by theohax** [comment]
Rebased this and included changes after on-line review session. 

@dsiganos @clemahieu please have a look guys, thanks

File: nano/core_test/conflicts.cpp [hunk 2]
```diff
@@ -85,145 +94,141 @@ TEST (conflicts, add_two)
...
+		auto const [send4, dummy2] = do_send ((*receive2)->hash (), account2, account3, false);
+		ASSERT_TRUE (send4.has_value ());
+		ASSERT_EQ (5, node->ledger.cache.cemented_count);
+
+		// activate elections for the previous two send blocks (to account3) that we did not forcefully confirm
+		//
+		node->scheduler.activate (account3.pub, node->store.tx_begin_read ());
+		ASSERT_TIMELY (5s, node->active.election ((*send3)->qualified_root ()) != nullptr);
+		ASSERT_TIMELY (5s, node->active.election ((*send4)->qualified_root ()) != nullptr);
+<<<<<<< HEAD
```
Comment (dsiganos) [inline nano/core_test/conflicts.cpp:139]: looks like the rebase went wrong

**Comment by theohax** [comment]
Looks like this PR got superseded by another one, test changes are already in `develop` and working, therefore closing this one

**Pull request closed without merging**